tests/scripts/all.sh

```diff
@@ -1886,13 +1925,18 @@ run_component () {
     # Unconditionally create a seedfile that's sufficiently long.
     # Do this before each component, because a previous component may
     # have messed it up or shortened it.
-    dd if=/dev/urandom of=./tests/seedfile bs=64 count=1
+    dd if=/dev/urandom of=./tests/seedfile bs=64 count=1 >/dev/null 2>&1
```

Comment:
Please don't redirect stderr in non-quiet mode. If an error occurs (e.g. disk full), we need to see the error message, at least in non-quiet mode. You don't need to redirect stdout since `dd` won't print anything there.